python/dgl/data/chem/README.md

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
We should not expect new users to easily find this file. For the time being, I'd suggest adding a line in `examples/pytorch/model_zoo/property_prediction/README.md` so that people can find it.

Reply:
I expect all these things will be moved to documentation later. Currently it's just for simplicity so I use markdown